fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1064,6 +1064,49 @@ void assertResolutionStateMutationsSizeConsistent(const std::vector<ResolveTrans
 	}
 }
 
+inline int64_t extractTenantIdFromKeyRef(StringRef s) {
```

Comment:
Given these methods are not specific to CP and also it is used in other places such as BackupActor, we might want to move it to a common util file if possible

Reply:
I moved this to the tenantManagement actor so we can use it in other places